pkg/api/pin_bytes_test.go

```diff
@@ -123,21 +124,41 @@ func TestPinBytesHandler(t *testing.T) {
 		)
 
 		hashes := []string{rootHash, data1Hash, data2Hash}
+		sort.Strings(hashes)
 
-		expectedResponse := api.ListPinnedChunksResponse{
-			Chunks: []api.PinnedChunk{},
-		}
+		// NOTE: all this because we cannot rely on sort from response
 
-		for _, h := range hashes {
-			expectedResponse.Chunks = append(expectedResponse.Chunks, api.PinnedChunk{
-				Address:    swarm.MustParseHexAddress(h),
-				PinCounter: 1,
-			})
-		}
+		var respBytes []byte
 
 		jsonhttptest.Request(t, client, http.MethodGet, pinChunksResource, http.StatusOK,
-			jsonhttptest.WithExpectedJSONResponse(expectedResponse),
+			jsonhttptest.WithPutResponseBody(&respBytes),
```

Comment:
unless you need the actual bytes for something, you can just use `jsonhttptest.WithUnmarshalJSONResponse` and unmarshal directly to `api.ListPinnedChunksResponse`